db/app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const { getTopics, invalidRoutes } = require("../controlers/topic-controler");
+const { customErrorHandler } = require("../errors");
+const app = express();
+app.use(express.json());
```

Comment:
This should also not be added at this point as it is not needed for this ticket.